CHANGELOG.md

```diff
@@ -4,7 +4,7 @@
 
 ### Grafana Mimir
 
-* [CHANGE] Querier: returning 422 when query hits `max_fetched_chunks_per_query` and `max_fetched_series_per_query` limits in the store-gateway. #4056
+* [CHANGE] Querier: returning 422 when query hits `max_fetched_chunks_per_query` and `max_fetched_series_per_query` limits either in the store-gateway or in querier. #4056 #4100
```

Comment:
Can you please explain what has changed for this [CHANGE]?

Reply:
Hi @pstibrany, 
Until this change querier was, probably erroneously, reporting a status check error in the case the limit was hit in the store-gateway. There are 2 PRs for the following reason:
- PR 4056: when a limit is hit in Limiter, an error with the status code 422 is created and returned. These errors are the ones that the store-gateway returns to the querier
- PR 4100: the status code 422 returned by the store-gateway is further propagated by the querier, and not converted to a consistency check anymore

Reply:
Would you please update CHANGELOG file to explain that? Right now the changelog entry says `[CHANGE]` but doesn't say what has changed, it only mentions what the code does now. As a user reading this entry, I don't understand why it's under `[CHANGE]`.

Reply:
Sure, I have updated CHANGELOG